network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/newsletter_signup_block.html

```diff
@@ -0,0 +1,21 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags static i18n l10n %}
+{% block main_block_class %}{% endblock %}
+
+{% block block_content %}
+
+<div class="-tw-ml-1 tw-mt-3 tw-mb-7 tw-py-5 tw-px-5 tw-rounded-lg tw-popout medium:tw-ml-0 medium:tw-px-7 medium:tw-pb-7">
+    <div
+      class="join-us"
+      data-form-position="callout-box"
+      data-form-style="pop"
+      data-layout="2-column"
+      data-signup-id="{{ self.signup.id | unlocalize }}"
+      data-cta-header="{{ self.signup.header | escape }}"
+      data-cta-description="{{ self.signup.description | escape }}"
+      data-newsletter="{{ self.signup.newsletter }}"
+      data-ask-name="{{self.signup.ask_name}}">
+    >
```

Comment:
Can you add some context how this works? I am guessing there is some JS uses this?